### ISSUE TITLE
Failed to find schema examples

### ISSUE THREAD

**Issue by radver** [bobby-brennan/fhir-swagger#11]
Hello,

Whenever I execute the fhir-seagger \ command I get a lot of errors saying:

Failed to find schema example for <e.g. ActivityDefinition>

I'm not sure which file from the package do i need to modify so that the tool can generate Swagger for fhir json files.

Thank you!

**Comment by rbren** [comment]
Hi there,

You can add example schemas here: https://github.com/bobby-brennan/fhir-swagger/tree/master/schemas

The schemas that are there came from [this page](https://www.hl7.org/fhir/downloads.html). It's possible they've added new schemas since the last download. Happy to accept a PR with ActivityDefinition or any other schema examples.

**Comment by radver** [comment]
Hello,

Thank you so much for the reply. Unfortunately, the application is giving me ETIMEOUT error now:

[cid:image001.png@01D4A285.C7E3EB60]

Any suggestions to resolve this?

Also, is it possible to start with few resources, instead of all of them? I want to convert say 4-5 JSON files (such as Patient, Practitioner, Organization, etc. ) to Swagger format. Not sure if the tool will allow, if yes, next question is how and where?

Thank you in advance for looking into the issues. Appreciate your response.

Kind regards,
Radhika

**Comment by radver** [comment]
Here is the description of the error message:

C:\Users\[username]\AppData\Roaming\npm\node_modules\fhir-swagger\bin\fhir-swagger:7
  if (err) throw err;
           ^

Error: connect ETIMEDOUT 54.87.74.90:80
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)

**Comment by rbren** [comment]
Sounds like that URL (`54.87.74.90:80`) is either unavailable or unresponsive.

There's currently no option to do a partial conversion, but I'd accept a PR that adds it.

**Comment by radver** [comment]
Thank you! Appreciate your help and response.

**Comment by thril** [comment]
Hi. I am getting the same error with a list of missing examples. However, I see that (e.g.) ActivityDefinition is in the examples...

https://github.com/bobby-brennan/fhir-swagger/blob/master/schemas/r4/examples/activitydefinition-example.json

Do you know why there's a discrepancy between it being in the repo and it not being able to find it?

**Comment by thril** [comment]
Dug into this a little bit more. I was able to get more recent definitions by specifying the latest commit.

```
"fhir-swagger": "https://github.com/bobby-brennan/fhir-swagger#666e26b5ce8738b99deaa6406315448c3dab679e"
```

Looks like the latest needs to be included in the next version. Thoughts on when that will be? Thanks!

**Comment by rbren** [comment]
Published as `2.3.0`, thanks!

**Comment by jkalinowskizajdak** [comment]
I still have get error: 
```
C:\Users[username]\AppData\Roaming\npm\node_modules\fhir-swagger\bin\fhir-swagger:7
if (err) throw err;
^

Error: connect ETIMEDOUT 54.87.74.90:80
at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)
```
example usage: 
fhir-swagger fhir_url "http://localhost:8084/api/fhir/" conformance_path="/metadata?_format=application/json" r4 output swagger.json 

@rbren do you know error reason?
